### PR TITLE
Update keybinds.lua

### PR DESCRIPTION
**Pull request by Martian-dev**
Changed the BufferClose command to `:bd`, cuz bufferClose wont work it isnt the correct command ;)

### CODE REVIEW

File: lua/keybinds.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
-vim.api.nvim_set_keymap("n", "<C-Q>", ":BufferClose<CR>", { noremap = true })
+vim.api.nvim_set_keymap("n", "<C-Q>", ":bd<CR>", { noremap = true }) -- BufferClose isnt the correct command to close a buffer ;)
```
Comment (classPythonAddike) [inline lua/keybinds.lua]: Can you please remove that comment :peep:, other than that, its fine!

**Review by classPythonAddike — CHANGES_REQUESTED** [review]

File: lua/keybinds.lua (hunk 1 above)
Comment (Martian-dev) [reply lua/keybinds.lua]: you can remove it urself :ugundu:

**Comment by Martian-dev** [comment]
Check it out now addike

**Review by classPythonAddike — APPROVED** [review]
lgtm

**Comment by classPythonAddike** [comment]
thanks for contributing @vaibhav-75007 :dorime:

**Pull request merged**